base-template.sh

```diff
@@ -14,13 +14,14 @@
 #   0 when successfully finished, 1 otherwise
 #####################################################
 process_git_hook() {
+    set_main_variables
+    register_for_autoupdate_if_needed
```

Comment:
Should we register this repo if Githooks is disabled here? :thinking: 
I think we should leave it as it was perhaps?